desktop/apps/consignments2/client/actions.js

```diff
@@ -357,17 +417,37 @@ export function selectPhoto (file) {
   }
 }
 
+export function startLoading () {
```

Comment:
They're dispatching `action`s (in this case `'START_LOADING'`) that are picked up by the reducers (in `reducers.js`). This one in particular will cause the `loading` state to change to `true` which is then picked up in the react views to show a spinner.